interfaces/builtin/cpu_control.go

```diff
@@ -35,6 +35,24 @@ const cpuControlConnectedPlugAppArmor = `
 /sys/devices/system/cpu/cpu*/online rw,
 /sys/devices/system/cpu/smt/*       r,
 /sys/devices/system/cpu/smt/control w,
+
+# https://www.kernel.org/doc/html/latest/admin-guide/pm/cpufreq.html#policy-interface-in-sysfs
+/sys/devices/system/cpu/cpufreq/* r,
+/sys/devices/system/cpu/cpufreq/policy*/* r,
+/sys/devices/system/cpu/cpufreq/policy*/energy_performance_preference w,
+/sys/devices/system/cpu/cpufreq/policy*/scaling_governor w,
+/sys/devices/system/cpu/cpufreq/policy*/scaling_max_freq w,
+/sys/devices/system/cpu/cpufreq/policy*/scaling_min_freq w,
+/sys/devices/system/cpu/cpufreq/policy*/scaling_setspeed w,
+/sys/devices/system/cpu/cpufreq/boost w,
+
+# https://www.kernel.org/doc/html/latest/admin-guide/pm/intel_pstate.html#user-space-interface-in-sysfs
+/sys/devices/system/cpu/cpufreq/intel_pstate/* r,
```

Comment:
I think this should be `/sys/devices/system/cpu/intel_pstate/`

Reply:
This makes every file there readable and select files are made writable by the rules below.

Reply:
No @bboozzoo is right - I added an extra 'cpufreq' into the path which should be removed - will repush a fixed version soon.